support-frontend/app/views/subscriptionCheckout.scala.html

```diff
@@ -53,6 +53,7 @@
           uat: "@uatStripeConfig.forCurrency(Some(AUD)).publicKey"
         }
       };
+      window.guardian.stripeKeyUnitedStates = window.guardian.stripeKeyDefaultCurrencies,
```

Comment:
What does this do @paulbrown1982

Reply:
It forces the US public key to be the same as the default one, because the new account won't be used for Subscriptions - it's easier to do this than to carry the 'what-product-is-it?' logic into the simple 'what country is it?' decision system about what Stripe account to use.